### DISCUSSION TITLE
Should we rename this library react-router-auth?

### DISCUSSION THREAD

**Discussion by bluefire2121** [sergiodxa/remix-auth#311] [Ideas]
In support of migrating remix to RRv7, should this library (and similar libraries) be renamed `react-router-auth` ?

**Comment by sergiodxa** [comment]
It's too long, and also it's already used on npm https://www.npmjs.com/package/react-router-auth

Also Remix Auth v4 works on Remix and RR.

**Reply by bluefire2121** [reply]
Not much we can do about the longer name, but maybe the author would be willing to give up the library. Doesn't hurt to ask :-) . I'm happy to reach out to him if you'd like @sergiodxa

**Reply by bluefire2121** [reply]
I've reached out to the owner of `react-router-auth` and he has graciously agreed to offer the name to the React Router team. Can we move forward to acquire the library?
